runtime/src/bank/serde_snapshot.rs

```diff
@@ -603,7 +585,8 @@ mod tests {
                 AccountsHash(Hash::new_unique()),
                 &get_storages_to_serialize(&snapshot_storages),
                 ExtraFieldsToSerialize {
-                    lamports_per_signature: bank.fee_rate_governor.lamports_per_signature,
+                    lamports_per_signature: solana_sdk::fee_calculator::FeeRateGovernor::default()
+                        .lamports_per_signature,
```

Comment:
`bank` is created as `default_for_tests()`, effectively same as `FeeRateGovernor::default().lamports_per_signature`.

Reply:
do we not have this constant anywhere else? if we don't maybe we just add it in `fee` crate instead of continue to use `FeeRateGovernor` code.